### ISSUE TITLE
optimizeForReactDom/Native does not work in IE11 due to destructuring

### ISSUE THREAD

**Issue by Tarpsvo** [mobxjs/mobx-react-lite#238]
Hi!

As the `optimizeForReactDom`and `optimizeForReactNative` opt-ins are not transpiled and use object destructuring, they do not work in certain browsers (namely, IE11).

Can we avoid destructuring and just do it a bit more conventionally?

For example:
```
const ReactDOM = require('react-dom');
require('./dist').optimizeScheduler(ReactDOM.unstable_batchedUpdates);
```

**Comment by FredyC** [comment]
Thanks for the report, you are right. Are you willing to provide PR?

**Comment by Tarpsvo** [comment]
Definitely, provided a PR at #239.

**Comment by FredyC** [comment]
Thank you. Can you also make the same PR to the `next` branch, please? It might be released soon as 2.0.

**Comment by Tarpsvo** [comment]
Thanks! PR opened at #240.

**Comment by FredyC** [comment]
I thank you :) Published 1.5.1.

**Comment by Tarpsvo** [comment]
No problem! Thanks for the quick reply, merge and new release!
